pilot/pkg/config/kube/gateway/deploymentcontroller.go

```diff
@@ -183,7 +192,20 @@ func getClassInfos() map[gateway.GatewayController]classInfo {
 			defaultServiceType: corev1.ServiceTypeClusterIP,
 			// Report both. Consumers of the gateways can choose which they want.
 			// In particular, Istio across different versions consumes different address types, so this retains compat
-			addressType: "",
+			addressType:     "",
+			controllerLabel: constants.ManagedGatewayMeshControllerLabel,
+		}
+	}
+
+	if features.EnableAmbientMultiNetwork {
+		m[constants.ManagedGatewayEastWestController] = classInfo{
+			controller:         constants.ManagedGatewayEastWestController,
+			description:        "The default GatewayClass for Istio East West Gateways",
+			templates:          "waypoint",
```

Comment:
Probably not good to share them especially now that we have customization. Given they are different classes I think users would expect them to have different customizations?

Reply:
actually NVM, that is based on the class not the template.

Reply:
I could go either way on this; sharing let me get something working really quickly, and felt like it would prevent staleness (e.g. we add some stuff to waypoints but not e/w gateways that are essentially waypoints). Agree on customization not really being a factor